src/features/defi/contexts/DefiManagerProvider/DefiCommon.ts

```diff
@@ -17,6 +18,44 @@ export enum DefiProvider {
   ThorchainSavers = 'THORChain Savers',
 }
 
+export const DefiProviderMetadata = {
+  [DefiProvider.Idle]: {
+    type: DefiProvider.Idle,
+    icon: IdleFinanceLogo,
+    name: 'Idle Finance',
+  },
+  [DefiProvider.Yearn]: {
+    type: DefiProvider.Yearn,
+    icon: 'https://rawcdn.githack.com/trustwallet/assets/master/blockchains/ethereum/assets/0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e/logo.png',
+    name: 'Yearn Finance',
+  },
+  [DefiProvider.ShapeShift]: {
+    type: DefiProvider.ShapeShift,
+    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    name: 'ShapeShift',
+  },
+  [DefiProvider.FoxFarming]: {
+    type: DefiProvider.FoxFarming,
+    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    name: 'FOX Farming',
```

Comment:
Note that this will be split between `UniV2` and `EthFoxStaking` providers and we'll need two entries for that now